pyatlan/client/atlan.py

```diff
@@ -133,6 +133,7 @@ def get_session():
             "x-atlan-agent": "sdk",
             "x-atlan-agent-id": "python",
             "User-Agent": f"Atlan-PythonSDK/{VERSION}",
+            "X-Atlan-Client-Origin": "product_sdk",
```

Comment:
Let's add the custom header above `User-Agent`. I would also change it to lowercase to ensure consistency with other custom headers, as [HTTP headers are case-insensitive:](https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive):

```suggestion
            "x-atlan-client-origin": "product_sdk",
            "User-Agent": f"Atlan-PythonSDK/{VERSION}",
```